lab-kayla/test/server-test.js

```diff
@@ -1,134 +1,135 @@
 'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Need the following tests:

GET - test 404, responds with 'not found' for valid request made with an id that was not found
GET - test 200, response body like {<data>} for a request made with a valid id
POST - test 400, responds with 'bad request' for if no body provided or invalid body
POST - test 201, response body like {<data>} for a post request with a valid body
PUT - test 400, responds with 'bad request' for if no body provided or invalid body
PUT - test 202, response body like {<data>} for a put request with a valid id
DELETE - test 404, responds with 'not found' for valid request made with an id that was not found
DELETE - test 204, response for a delete request with a valid id